src/coreservices.cpp

```diff
@@ -495,6 +495,8 @@ void CoreServices::initialize(QApplication* pApp) {
 
     m_isInitialized = true;
 
+    ControllerScriptEngineBase::registerPlayerManager(getPlayerManager());
```

Comment:
What do you think about not using a singleton here since you don't have the QML requirement to do so? ?

Reply:
Wait, how would I do that? How can I retrieve the `PlayerManager` from `JavascriptPlayerProxy` ?